addons/cluster-autoscaler/cluster-autoscaler.yaml

```diff
@@ -28,6 +28,9 @@
 {{ if eq .Cluster.MajorMinorVersion "1.31" }}
 {{ $version = "v1.31.0" }}
 {{ end }}
+{{ if eq .Cluster.MajorMinorVersion "1.32" }}
+{{ $version = "v1.32.0" }}
+{{ end }}
```

Comment:
Please run `make` in the cluster-autoscaler directory to rebuild the YAML.

Reply:
done!